packages/time-picker/test/events.test.js

```diff
@@ -105,6 +105,46 @@ describe('events', () => {
       inputElement.blur();
       expect(changeSpy.callCount).to.equal(1);
     });
+
+    it('should not be fired again on Enter if the value has not changed', async () => {
+      inputElement.focus();
+      await sendKeys({ type: '10:00' });
+      await sendKeys({ press: 'Enter' });
+      expect(changeSpy).to.be.calledOnce;
```

Comment:
Let's change chai assertions:

```suggestion
      expect(changeSpy.calledOnce).to.be.true;
```